src/app/dynamic/enterprise/metering/module.ts

```diff
@@ -22,21 +22,38 @@ import {NgModule} from '@angular/core';
 import {RouterModule, Routes} from '@angular/router';
 import {MeteringComponent} from '@app/dynamic/enterprise/metering/component';
 import {MeteringConfigComponent} from '@app/dynamic/enterprise/metering/config/component';
+import {MeteringScheduleConfigComponent} from '@app/dynamic/enterprise/metering/schedule-config/component';
 import {MeteringConfigurationDialog} from '@app/dynamic/enterprise/metering/config/config-dialog/component';
 import {MeteringCredentialsDialog} from '@app/dynamic/enterprise/metering/config/credentials-dialog/component';
-import {MeteringListComponent} from '@app/dynamic/enterprise/metering/list/component';
+import {MeteringScheduleAddDialog} from '@app/dynamic/enterprise/metering/schedule-config/add-dialog/component';
+import {MeteringScheduleEditDialog} from '@app/dynamic/enterprise/metering/schedule-config/edit-dialog/component';
+import {MeteringReportListComponent} from '@app/dynamic/enterprise/metering/schedule-config/report-list/component';
 import {SharedModule} from '@shared/module';
+import {AuthGuard, AuthzGuard} from '@core/services/auth/guard';
 
-const routes: Routes = [{path: '', component: MeteringComponent}];
+const routes: Routes = [
+  {
+    path: '',
+    component: MeteringComponent,
+  },
+  {
+    path: 'schedule/:scheduleId',
+    component: MeteringReportListComponent,
+    canActivate: [AuthGuard, AuthzGuard],
```

Comment:
Remove this. `AuthzGuard` is used only for project-related resources when the user is inside the project. `AuthGuard` is already defined globally for the whole admin settings routing. No need to add it here.